app/auth/identity.py

```diff
@@ -63,7 +63,8 @@ def __init__(self, obj=None, token=None):
 
             if obj["type"] == IdentityType.USER:
                 self.identity_type = obj["type"]
-                self.user = obj["user"]
+                if obj.get("user"):
+                    self.user = obj["user"]
```

Comment:
It's simpler to just do:
```suggestion
                self.user = obj.get("user")
```